scripts/ci/upload-universe

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash#
```

Comment:
is the trailing `#` intended?

Reply:
Shall we hold a meeting to decide on a language for scripts?

Reply:
Nope and Nope, you are right, this shall be done in JS. Will rework it 👍